### PR TITLE
feat: Edit metadata

### PR DESCRIPTION
**Pull request by Predjo**
## Motivation

Add a modal to change Amity metadata

## Testing Done

- [ ] I have successfully ran the unit tests
- [ ] I have successfully ran the integration tests
- [ ] I have successfully completed the manual testing of my changes and the surrounding code

## Screenshots
Screen Shot 2023-02-03 at 4 28 30 PM

### CODE REVIEW

File: src/social/components/Feed/index.js [hunk 1]
```diff
@@ -34,6 +34,11 @@ function getAdjustedPostTarget(
   return { targetType, targetId };
 }
 
+function reloadPage() {
+  // TODO: Replace with proper feed replace
```
Comment (Bristina) [inline src/social/components/Feed/index.js:38]: did you forget this or will you do it in another PR?

File: src/social/components/UserInfo/UserMetadata.tsx [hunk 2]
```diff
@@ -26,95 +45,199 @@ type NoomMetadata = {
...
+};
 
 export const UserMetadata = ({ user }: { user: User }) => {
-  const noomMetadata = useNoomUserMetadata(user.userId) as NoomMetadata;
-  return <UIUserMetadata user={user} noomMetadata={noomMetadata} />;
+  const { metadata, updateMetadata } = useNoomUserMetadata(user.userId);
+
+  const handleUpdate = (userAccessCode: string, metadata: any) => {
+    updateMetadata(userAccessCode, metadata, () => {
+      //TODO: Replace this with actual fetch from Amity
```
Comment (Bristina) [inline src/social/components/UserInfo/UserMetadata.tsx:235]: also here

**Review by Bristina — APPROVED** [review]

File: src/social/components/Feed/index.js (hunk 1 above)
Comment (Predjo) [reply src/social/components/Feed/index.js:38]: I don't have time to do it. It is something for the future devs.
